server/i18n/en.json

```diff
@@ -8878,11 +8874,11 @@
   },
   {
     "id": "model.config.is_valid.elastic_search.enable_autocomplete.app_error",
-    "translation": "Search EnableIndexing setting must be set to true when Elasticsearch EnableAutocomplete is set to true"
+    "translation": "Search EnableIndexing setting must be set to true when {{.SettingName}} is set to true"
```

Comment:
I think `Search EnableIndexing` should also become a variable and become the actual setting name which is `ElasticsearchSettings.EnableIndexing`